src/components/GetStarted/EditPreferences.vue

```diff
@@ -181,12 +181,67 @@ export default {
 			default: 0,
 		},
 	},
+	data() {
+		return {
+			causeList: [
```

Comment:
This is repeated in `src/pages/GetStarted/GetStartedCauses.vue` and could be moved to a data mixin like the one we have for `src/plugins/loan-channel-query-map.js`

Reply:
True, just not a priority at this time.